CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# The Rust Code of Conduct
+
+## Conduct
+
+**Contact**: [Embedded Linux Team][team]
+
+* We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
```

Comment:
Could make this 80 columns wide like the rest of the document 🤷‍♂️ . It makes it easier to read as plain text at least.